src/test_mailroom.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+TEST_DICT = {
+    ('lessor', 'crystal'): [10, 40, 25],
+    ('harrison', 'mike'): [30, 40, 15],
+    ('smith', 'john'): [10, 25],
+    ('python', 'monty'): [42]
+}
+
+donor_list = {
+    ('lessor', 'crystal'): [10, 40, 25],
+    ('harrison', 'mike'): [30, 40, 15],
+    ('smith', 'john'): [10, 25],
+    ('python', 'monty'): [42],
+}
+
+TEST_LIST = [
+    (('harrison', 'mike'), [30, 40, 15], 28.333333333333332),
+    (('lessor', 'crystal'), [10, 40, 25], 25.0),
+    (('python', 'monty'), [42], 42.0),
+    (('smith', 'john'), [10, 25], 17.5)
+]
+
+
+def test_validate_first_0():
+    """Test validation for first name."""
+    from mailroom import validate_first_name
+    assert validate_first_name('mike') == 'mike'
```

Comment:
The way that `validate_first_name` handles invalid input means you can't test what happens when the input is invalid.  That's not so good.  The same applies for `validate_last_name` below.